api.go

```diff
@@ -234,6 +234,16 @@ type (
 		Status TaskStatus `json:"status"`
 	}
 
+	importAssetRequest struct {
+		Name string `json:"name,omitempty"`
+		URL  string `json:"url"`
+	}
+
+	ImportAssetResp struct {
```

Comment:
perhaps consistent naming pattern: `Resp` vs `Response` (similar to `Request` above)?